builtin/provisioners/chef/linux_provisioner.go

```diff
@@ -11,7 +11,7 @@ import (
 
 const (
 	chmod      = "find %s -maxdepth 1 -type f -exec /bin/chmod %d {} +"
-	installURL = "https://www.chef.io/chef/install.sh"
+	installURL = "https://omnitruck.chef.io/chef/install.sh"
```

Comment:
Drop the `chef` in this URL...we just want to use `https://omnitruck.chef.io/install.sh`, more details at https://docs.chef.io/packages.html#omnitruck-install-script-options